<launch>
    <node pkg="pointcloud_utils" type="pointcloud_saver" name="pointcloud_saver"
          required="true" output="screen">
        <param name="cloud_topic" 		value="/velodyne_points" />
        <param name="filename_base" 	value="$(find pointcloud_utils)/points/points" />
        <param name="file_extension" 	value=".csv"   />

        <param name="parse_from_bag" value="true"/>
        <param name="bagfile" value="/home/winston/Documents/ws/280_hghwy_cut_2021-03-28-18-21-35.bag" />
    </node>
</launch>